src/scenes/components/Speakers/Speakers.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import AppContainer from '../../../components/AppContainer/AppContainer'
+import { H2 } from '../../../components/typography'
+import Speaker from './components/Speaker/Speaker'
+
+const Root = styled.section`
+  padding-top: 80px;
```

Comment:
We are using a lot of fixed values on `padding` and `margin` do you think it would make sense to create a a mixin to have these spacing normalized, like only multiples of 4 or 5, something like:

```
const SPACING_CONSTANT = 5;

const spacingMixin = multiplier => return multiplier * SPACING_CONSTANT 
```

Anyway, just some food for thought

Reply:
We should indeed introduce such normalization, but for that we'll have to discuss it with the design. Will do it.